guidelines/design-document-template.md

```diff
@@ -0,0 +1,35 @@
+# Design Document - My proposal 
+
+Author : **myemail@email.com**
+Date : Oct 2020
+
```

Comment:
```suggestion
```

We don't need these duplicated in the doc, as they're rather directly available from the edit history.